packages/react/src/withComponentStack.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import ReactSharedInternals from 'shared/ReactSharedInternals';
+
+function noop() {}
+
+let error = noop;
+let warn = noop;
+if (__DEV__) {
+  const ReactDebugCurrentFrame = ReactSharedInternals.ReactDebugCurrentFrame;
+
+  error = (...args) => {
```

Comment:
Let's write out the ES5 code. I don't trust the Babel plugin to do a better job than we do.